plugins/com.google.cloud.tools.eclipse.sdk.ui/src/com/google/cloud/tools/eclipse/sdk/ui/preferences/messages.properties

```diff
@@ -6,5 +6,5 @@ CloudSdkPreferencePage_4=Cannot launch a browser
 CloudSdkPreferencePage_5=&SDK location:
 CloudSdkPreferencePage_6=No SDK found: {0}
 CloudSdkPrompter_0=Google Cloud SDK Not Configured
-CloudSdkPrompter_1=The Google Cloud SDK cannot be found\!  Would you like to configure it now?
+CloudSdkPrompter_1=Invalid or no Google Cloud SDK specified. Do you want to configure Google Cloud SDK now?
```

Comment:
How about:

The Google Cloud SDK is required but cannot be found\!  Would you like to configure it now?